ws_sdk/app.py

```diff
@@ -727,6 +738,24 @@ def get_products(self,
                                                sort_by=sort_by)
         return products
 
+    def get_product_by_name(self,
```

Comment:
Check [get_scopes_from_name()](https://github.com/whitesource-ps/ws-sdk/blob/6cd8fb95a8197169092d7fb057babbf854ca5b09/ws_sdk/app.py#L673) to be used instead.

Reply:
Yes but that returns a list. In this helper function you get only one result

Reply:
Hi @meramsey - it returns a list for cases where you might have more than 1 entity with the same name :
1. Global organization --> products with the same name in different organizations.
2. Non-global organization--> projects with the same name within different products.